tracer/test/snapshots/ProcessStartTests.SubmitsTraces.verified.txt

```diff
@@ -0,0 +1,122 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: command_execution,
+    Resource: nonexisting1.exe,
+    Service: Samples.ProcessStart-command,
+    Type: system,
+    Error: 1,
+    Tags: {
+      env: integration_tests,
+      error.type: System.ComponentModel.Win32Exception,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: internal,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_3,
+    SpanId: Id_4,
+    Name: command_execution,
+    Resource: nonexisting2.exe,
+    Service: Samples.ProcessStart-command,
+    Type: system,
+    Error: 1,
+    Tags: {
+      env: integration_tests,
+      error.type: System.ComponentModel.Win32Exception,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: internal,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_5,
+    SpanId: Id_6,
+    Name: command_execution,
+    Resource: nonexisting3.exe,
```

Comment:
One thing I noticed is that the Linux test doesn't produce spans for `nonexisting3.exe` and `nonexisting4.exe`. Do you know why that happens?

Reply:
Because the overloads of start() used to launch these processes are not allowed in Linux.

Reply:
Cool, thanks 😄